cabal-install/Distribution/Client/Setup.hs

```diff
@@ -657,12 +696,26 @@ buildExOptions _showOrParseArgs =
 
 buildCommand :: CommandUI (BuildFlags, BuildExFlags)
 buildCommand = parent {
+    commandName = "build",
+    commandDescription  = Just $ \_ -> wrapText $
+      "Components encompass executables, tests, and benchmarks.\n"
+        ++ "\n"
+        ++ "Affected by configuration options, see `v1-configure`.\n",
```

Comment:
I think the convention is to use straight quotes? This occurs a bunch so you'll have to search-and-replace.

Reply:
That's just copy-pasted from `Distribution.Simple.Setup` so if it's inconsistent that's not on me. 😛

Reply:
Oh, fair point. I do see we're inconsistent today. I've opened #5377 as a TODO item about it.